### DISCUSSION TITLE
Unter-Ordner in Favoriten anlegen. / Create sub-folders in favorites.

### DISCUSSION THREAD

**Discussion by 9Biker** [osmandapp/OsmAnd#18875] [General]
Wie legt man Unter-Ordner in Favoriten an.

How to create subfolders in favorites.

**Comment by pebogufi** [comment]
You can create groups in favorites. 
When editing a fav, just move the group selection to the very right position, there you find "add a group".

**Comment by 9Biker** [comment]
Vielen Dank!
